src/lib/custom-resources/cdk-opensearch-siem-configure/README.md

```diff
@@ -0,0 +1,23 @@
+# Put S3 Bucket Notifications
```

Comment:
To the top of this readme.md can we add something to the effect:
# siem-on-amazon-opensearch-service
The AWS Secure Environment Accelerator now supports deploying the AWS Open Source project aws-samples/siem-on-amazon-opensearch-service which can be found here: https://github.com/aws-samples/siem-on-amazon-opensearch-service.  The current copies of the files in this folder are based on siem-on-amazon-opensearch-service
 v2.6.0 (tag) and will periodically be updated.
- os-loader.zip - can be found here: {path to file}
- opensearch-config.json - can be found here: {path to file}
- dashboard.ndjson.zip - can be found here: {path to file}
(If the file is not found directly on the os project, provide the specific instructions to create it from that project)

Reply:
I enhanced this readme with your feedback